cmd/run.go

```diff
@@ -37,28 +36,39 @@ attach to a defined IP and ports with given options. This mode is ideal for test
 with the underlying DNS server. Use this mode for testing, debugging and benchmarks.
 		`,
 		RunE: func(cmd *cobra.Command, args []string) error {
+
+			allSettings := runViper.AllSettings()
+			builtInHosts := allSettings["built-in-hosts"]
+			hosts, err := cast.ToStringMapStringE(builtInHosts)
+			if err != nil {
+				logrus.Errorf("reading built-in-hosts value: %v", err)
+			}
```

Comment:
```suggestion
			hosts := runViper.GetStringMapString("built-in-hosts")
```
Maybe I'm missing something?

Reply:
I tried, but it doesn't work. I think viper has issues with receiving maps through env-var.

Reply:
Odd; it seemed to work for me (or at least, the tests still pass). That was using `StaticHosts:     runViper.GetStringMapString("built-in-hosts"),`.
(I initially forgot to remove the `cast` import and the build failed, so I'm definitely making changes to the right file at least…)

Of course, there's also the chance that the test isn't working correctly, which is an issue too.

Reply:
I will investigate again.